react/src/components/dropdown/SprkDropdown.test.js

```diff
@@ -25,30 +25,48 @@ describe('SprkDropdown:', () => {
     expect(wrapper.find('.sprk-c-Dropdown__footer').length).toBe(1);
   });
 
-  it('should add classes to the dropdown when additionalClasses has a value', () => {
```

Comment:
can you revert the style changes to the test file and leave only the added tests?

Reply:
had done these changes because eslint was breaking as a part of git pre-commit  hook

Reply:
Ahh ok, I'll take a look

Reply:
You're right! Thanks for fixing those line lengths as part of this task!